pkg/bot/lark.go

```diff
@@ -183,10 +188,7 @@ func (l *LarkBot) SayHello(e map[string]interface{}) error {
 	}
 	users, ok := larkUserList.([]interface{})
 	if !ok {
-		user, ok := event[larkUsers].(interface{})
-		if !ok {
-			return fmt.Errorf("Invalid user format. Failed to convert user into interface{}")
-		}
+		user := event[larkUsers]
```

Comment:
Any reason behind removing this? We check if the user exists in the event map before adding it.

Reply:
There's no need to assert the type as it is already `interface{}`. That was the original issue reported by linter:
```
pkg/bot/lark.go:186:15: S1040: type assertion to the same type: event[larkUsers] already has type interface{} (gosimple)
		user, ok := event[larkUsers].(interface{})
```

If the intention was to check whether such user exists and return error if not, then I can fix it, let me know 🙂

Reply:
Yes, we should check if the user exists otherwise return error

Reply:
Right, sorry, I brought it back 👍